systems-management/nisysmgmt.yml

```diff
@@ -587,7 +587,25 @@ definitions:
         example: 19.0
     required: [activated]
 
-  ManagedSystemDataProperty:
+  ManagedSystemDataPropertyList:
+    description: Property of a system composed by the last updated timestamp and a list with the actual data.
+    type: object
+    properties:
+      lastUpdatedTimestamp:
+        description: Last updated timestamp.
+        type: string
+        format: iso-date-time
+        example: '2019-02-11T13:32:06.030958Z'
+      data:
+        description: The actual data.
+        type: array
+        items:
+          type: object
```

Comment:
So this can be an array of anything? There isn't any way to be more descriptive in the array's description for what clients can expect here (i.e. an array of scalars).

Reply:
This is for sysapi data - I replied above why we use 'object' here.

Reply:
Makes sense. Is there a way some of that information could be summarized in the description here?

Reply:
I did add better descriptions.